messages/training-checkout.txt

```diff
@@ -6,17 +6,31 @@ Thanks again for taking part in Software Carpentry (SWC) and Data Carpentry (DC)
 
 - If you want to teach SWC, you need to demonstrate that you've mastered one of its core lessons.  Similarly, if you want to teach DC, you need to show you've mastered one of its.  (This is Steps 2 and 3 below, which you do separately for each Carpentry.)
 
-The steps are therefore:
+The steps are:
 
-1. Submit a small improvement to one of our lessons.  If you want to become a SWC instructor, or to teach both SWC and DC, you must do this as a GitHub pull request; if you want to become a DC instructor only, go to http://www.datacarpentry.org/instructor-checkout-exercises/, which will redirect you to a Google Doc - please append your addition to the bottom of the doc to make it easier to find.  Either way, please mail me once your submission is in so I can get it reviewed.  You only have to submit *one* change to move on to the next step, not one per Carpentry, and we strongly prefer contributions to Data Carpentry right now (since their lessons are much younger).
+1. Submit an exercise or diagram for one of our lessons.  If you want to become a SWC instructor, or to teach both SWC and DC, you must do this as a GitHub pull request; if you want to become a DC instructor only, go to http://www.datacarpentry.org/instructor-checkout-exercises/, which will redirect you to a Google Doc - please append your addition to the bottom of the doc to make it easier to find.  Either way, please mail me once your submission is in so I can get it reviewed.  You only have to submit *one* change to move on to the next step, not one per Carpentry.
```

Comment:
At some point there should be a dedicated instructor training email so that we can retrieve these emails  when a given trainer is for any reason unavailable.